src/components/common/nav/TitleNavBar.tsx

```diff
@@ -56,7 +56,7 @@ function TitleNavBar({ title, subTitle = '', useBackIcon = true, children, onLef
   const navigate = useNavigate();
 
   return (
-    <Container>
+    <Container useSubTitle={subTitle !== ''}>
```

Comment:
```suggestion
    <Container useSubTitle={subTitle}>
```

이렇게 안되나요?? 안되면 !!subTitle도 있고 subTitle.isEmpty() 같은 것도 있을 것 같아요